### PR TITLE
docs: enable story source for current stories

### PR DESCRIPTION
**Pull request by rshen91**
## Summary

This PR enables the story source add-on for stories while consumer facing storybook is in progress. 

Visual regression tests were rerun because of the url naming changes with changing the fs for /stories/. 

The webpack config was edited to add story-source for each folder containing separate stories. Please feel free to propose a more succinct solution in the webpack config for the 14 or so folders (wildcard didn't work in testing). 

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.

~~- [ ] Any consumer-facing exports were added to `src/index.ts` (and stories only import from `../src` except for test data & storybook)~~
~~- [ ] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)~~
- [x] Proper documentation or storybook story was added for features that require explanation or tutorials
- [x] Unit tests were updated or added to match the most common scenarios
- [x] Each commit follows the [convention](https://github.com/elastic/elastic-charts/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by markov00** [comment]
Hey @rshen91 I've seen that we can maybe do a cleaner path instead of reenabling the info addon that bring in some issues: we can use the [StorySource addon](https://www.npmjs.com/package/@storybook/addon-storysource) that adds a new tab near the Knobs with the code listed directly from the story.
The minor drawback here is that, to avoid long waiting times caused by our very long story source files we should split the stories into multiple files (one per story) that is easy to read and display.
I've tested it out with the `scales` story that is the shortest one, and works fine. These are the passages I've done:

- add the addon: `yarn add @storybook/addon-storysource --dev`
- register it in `.storybook/addons.js` `import '@storybook/addon-storysource/register';`
- add the source loader for webpack (I've added it as the first rule, but not sure if this can be used in a different order:
```
config.module.rules.push({
    test: /\.tsx?$/,
    include: [path.resolve(__dirname, '../stories', 'scales')],
    loaders: [
      {
        loader: require.resolve('@storybook/source-loader'),
        options: { parser: 'typescript' },
      },
    ],
    enforce: 'pre',
  });
```
- I've created a `scales` folder and I've splitted each story in its own file, adding a `1_`, `2_` prefix on each file to order them as it was before


For me it seems to render a more readable and usable code than the addon-info
Screenshot 2020-02-19 at 11 04 57

File: docs/charts.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const Basic = () => {
     <Chart className={className}>
       <BarSeries
         id={specId}
-        name={'Simple bar series'}
+        name="Simple bar series"
```
Comment (nickofthyme) [inline docs/charts.tsx]: 😍❤️💝💕 

cry

File: integration/tests/annotations-stories.test.ts [hunk 2]
```diff
@@ -4,22 +4,22 @@ describe('Annotations stories', () => {
   describe('rotation', () => {
     it('rotation - 0', async () => {
       await common.expectChartAtUrlToMatchScreenshot(
-        'http://localhost:9001/?path=/story/annotations--test-line-annotation-single-value-histogram&knob-debug=&knob-chartRotation=0',
+        'http://localhost:9001/?path=/story/annotations-lines--single-bar-histogram&knob-debug=&knob-chartRotation=0',
```
Comment (nickofthyme) [inline integration/tests/annotations-stories.test.ts]: 👍

File: wiki/consuming.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 You can import Chart components from the top-level Elastic Chart module.
 
 ```js
-import { Axis, BarSeries, Chart, getAxisId, getSpecId, Position, ScaleType } from '@elastic/charts';
```
Comment (nickofthyme) [inline wiki/consuming.md]: 🎉

File: wiki/overview.md [hunk 4]
```diff
@@ -48,18 +48,18 @@ A spec can be something like the following:
 ```jsx
 <Chart renderer={renderer}>
   <Settings rotation={rotation} animateData={true} />
-  <Axis id={getAxisId('bottom')} position={AxisPosition.Bottom} title={`Rendering test`} />
-  <Axis id={getAxisId('left')} position={AxisPosition.Left} />
+  <Axis id={('bottom')} position={AxisPosition.Bottom} title={`Rendering test`} />
```
Comment (nickofthyme) [inline wiki/overview.md]: ??? no linting on mds 😞 

This can be fixed later

**Pull request merged**